### DISCUSSION TITLE
Is there's a way to have to achive the "fine detail mode" like XP Pen Driver.

### DISCUSSION THREAD

**Discussion by mufabil45** [OpenTabletDriver/OpenTabletDriver#3523] [Q&A] [answered]
![Screenshot_20241116_130812](https://github.com/user-attachments/assets/3e835d13-6597-435d-893c-5376de25adad)
In XP Pen there's a way to switch to the screen area with a toggle of a button. Any way to achieve this with OTD?

**Comment by jamesbt365** [comment, marked as answer]
Preset Bindings plugin or use precision control to "multiply" your area.

**Reply by mufabil45** [reply]
Thanks for helping me out. It works like I expected.
Though, I wonder, is there a way to at least display some text when the key is pressed?

Thanks in advance.

**Reply by gonX** [reply]
Making screen overlays is difficult and very platform-specific. It is unlikely that we'll get overlay displays, unless someone decides to tackle that.
